src/Microsoft.Health.Fhir.CosmosDb/Features/Health/CosmosHealthCheck.cs

```diff
@@ -64,6 +64,10 @@ public async Task<HealthCheckResult> CheckHealthAsync(HealthCheckContext context
 
                 return HealthCheckResult.Healthy("Successfully connected to the data store.");
             }
+            catch (CosmosException ex) when (ex.IsCmkClientError())
+            {
+                return HealthCheckResult.Healthy($"Connection to the data store was unsuccesful, however this is expected because the client's customer-managed key is not available. Error: {ex.Message}");
```

Comment:
Just following up here, did we decide to change this to Unhealthy?

Reply:
Yes.  I will change that in my next iteration.  Additionally, I plan on including more info in the health check to denote this is a client CMK error, I will includer the Exception itself, or a dictionary entry like "isClientCustomerManagedKeyError"=true (or something like that), or possibly both.  I'm leaning towards both.

Reply:
The dictionary entry will denote that the error has been classified as a client CMK error, and the exception will contain an error message saying what the error actually is.